source/templates/workflows.slim

```diff
@@ -135,11 +135,11 @@
 										.parameter.key
 											input[value="{{ output | prettifiedVariableKey }}" readonly]
 										.parameter.description
-											p[ng-if="!output.shortestDescription()"] == data[:strings][:workflows][:outputs][:no_description]
-											div[ng-if="output.shortestDescription()"]
+											p[ng-if="!output.title()"] == data[:strings][:workflows][:outputs][:no_description]
+											div[ng-if="output.title()"]
```

Comment:
If title is always available, there's no real point in keeping the `ng-if`. However, I've seen weirder things happen, so it might be the best to keep it as a safety net... 🤔 It's your call wether to keep it or not, I'm fine with it either way.